Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```
@@ -167,7 +167,7 @@
   - type: Item
     size: Huge
     shape:
-    - 0,0,7,2
+    - 0,0,8,2
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)